src/librustc_middle/mir/mono.rs

```diff
@@ -171,12 +170,12 @@ impl<'tcx> MonoItem<'tcx> {
         !tcx.subst_and_check_impossible_predicates((def_id, &substs))
     }
 
-    pub fn to_string(&self, tcx: TyCtxt<'tcx>, debug: bool) -> String {
+    pub fn to_string(&self, tcx: TyCtxt<'tcx>) -> String {
         return match *self {
-            MonoItem::Fn(instance) => to_string_internal(tcx, "fn ", instance, debug),
+            MonoItem::Fn(instance) => to_string_internal(tcx, "fn ", instance),
```

Comment:
Can't this just be `format!("fn {}", instance)`?

Reply:
In fact, I suspect `MonoItem` could just implement `fmt::Display`?

Reply:
Fixed!